Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -167,30 +163,44 @@ def create_revert_commit_msg(cls, args, commit_objects, **kwargs):
                 reverted_changeset += '\nhttps://commits.webkit.org/{}@{}\n'.format(commit.identifier, commit.branch)
             else:
                 sys.stderr.write('Could not find "{}"'.format(', '.join(args.commit)) + '\n')
-                return None
+                return None, None
 
         # Retrieve information for the issue tracking the revert
         revert_issue = Tracker.from_string(args.issue)
         if not revert_issue:
             sys.stderr.write('Could not find issue {}'.format(revert_issue))
-            return None
-        revert_issue_radar = None
-        for bug in CommitProgram.bug_urls(revert_issue):
-            for regex in cls.RES:
-                if regex.match(bug):
-                    revert_issue_radar = bug
+            return None, None
+
+        revert_radar_link, revert_bug_link = None, None
+        if isinstance(revert_issue.tracker, radar.Tracker):
+            revert_radar_link = revert_issue.link
+            refs = [i for i in revert_issue.references if isinstance(i.tracker, bugzilla.Tracker)]
+            if refs:
+                revert_bug_link = refs[0].link
+        else:
+            revert_bug_link = revert_issue.link
+            for bug in CommitProgram.bug_urls(revert_issue):
+                for regex in cls.RES:
+                    if regex.match(bug):
+                        revert_radar_link = bug
+                        break
+                if revert_radar_link:
                     break
-            if revert_issue_radar:
-                break
+
+        if revert_issue.redacted or isinstance(revert_issue.tracker, radar.Tracker):
```

Comment:
I think we should put a "FIXME" here. This is the correct behavior for some repositories, but that's really only true if `radar` isn't the primary tracker for the repository, right?